app/views/people/all_tasks.html.erb

```diff
@@ -0,0 +1,40 @@
+<%= render "shared/header" %>
+
+<div class="persons_tasks">
+
+  <p class="bold"> Finished Tasks </p>
+    <% @tasks.each do |task| %>
+      <% if task.completed_at? %>
+        <p> <%= task.name %> </p>
+      <% end %>
+    <% end %>
+<br>
+  <p class="bold"> Get It Done! </p>
+    <% @tasks.each do |task| %>
+      <% if task.completed_at == nil || task.completed_at == "" %>
+        <p> <%= task.name %> </p>
+      <% end %>
+    <% end %>
+  <!-- <table>
```

Comment:
I'd recommend going through you files and scrubbing them for extraneous commented out code when you submit your PRs